source/options.tsx

```diff
@@ -154,14 +154,22 @@ async function findFeatureHandler(event: Event): Promise<void> {
 }
 
 function summaryHandler(event: delegate.Event<MouseEvent>): void {
-	if (event.altKey || event.ctrlKey || event.metaKey || event.shiftKey) {
+	if (event.ctrlKey || event.metaKey || event.shiftKey) {
 		return;
 	}
 
 	event.preventDefault();
-	const feature = event.delegateTarget.parentElement!;
+	if (event.altKey) {
+		for (const screenshotLink of select.all('.screenshot-link')) {
+			toggleScreenshot(screenshotLink.parentElement!);
```

Comment:
Would it be too much code to wrap this in a IntersectionObserver a-la-`show-whitespace`?